dask_sql/context.py

```diff
@@ -248,9 +254,37 @@ def create_table(
         if statistics:
             self.schema[schema_name].statistics[table_name.lower()] = statistics
 
+        dc_statistics = None
+        if parquet_statistics and not statistics:
+            try:
+                stats = parquet_statistics(input_table)
+                if stats:
+                    row_count = 0
+                    for d in stats:
+                        row_count += d["num-rows"]
+                    dc_statistics = Statistics(row_count)
+            except ValueError:
+                if str(format).lower() == "parquet":
+                    if dask_cudf:
+                        ddf = dask_cudf.read_parquet(input_table)
+                    else:
+                        ddf = dd.read_parquet(input_table)
```

Comment:
In what case do we need to go down this branch?

Reply:
This handles if we were doing `c.create_table(table_name, my_filepath, format="parquet", gpu=True)` (instead of having a `read_parquet` then `c.create_table("ddf", ddf)`.